microsoft-edge/webview2/release-notes.md

```diff
@@ -26,6 +26,24 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
 
+## 1.0.1018-prerelease
+
+Release Date: September 20, 2021
+
+[NuGet package for WebView2 SDK 1.0.1018-prerelease][NuGetGallery1.0.1018-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 95.0.1018.0 or higher.
+
+### General
+
+#### Experimental Features
+*   Added [Media API][Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease] that enables developers to mute/unmute media within the webview.
```

Comment:
Are the links on these two lines going to be revised to go to mid-page sections?  The page that I arrive at (after deleting "review." from the 404 URL)  doesn't have a clear relationship to the link text here.
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1468#experimental-features

Reply:
@mikehoffms The API members on ICoreWebView2Experimental9 interface are various APIs to provide media control such as `IsMuted` and `IsDocumentPlayingAudio` properties. I believe "media" here is a generic term. Would `[media APIs](link)` be better in this context?

Reply:
Lowercased 'media' and linked to Summary # section which is clearly media-related unlike top of Ref page.